jscomp/bsb/bsb_global_paths.ml

```diff
@@ -66,4 +66,17 @@ let vendor_bsppx =
   
 ;; assert (Sys.file_exists bsc_dir)       
 
+let version = begin
+  let version = Sys.ocaml_version in 
+  match String.rindex version '+' with
+    | exception Not_found -> version
+    | i ->
+        String.sub version 0 i
+  end
```

Comment:
This looks good to me.
Can we hard code the version to shave some time since it would take a while for us to upgrade to new version

Reply:
Done!